tests/collections/collection.test.ts

```diff
@@ -373,6 +438,20 @@ describe(`StargateMongoose - ${testClientName} Connection - collections.collecti
             assert.strictEqual(findResDocs.length, 1);
             assert.strictEqual(findResDocs[0]._id, idToCheck);
         });
+        it('should find & findOne L1 Null NE $ne document', async () => {
+            const doc1 = createSampleDocWithMultiLevel();
+            const doc2 = createSampleDoc2WithMultiLevel();
+            const insertDocResp1 = await collection.insertOne(doc1);
+            const insertDocResp2 = await collection.insertOne(doc2);
+            const idToCheck2 = insertDocResp2.insertedId;
+            const filter = { 'password': { '$ne': doc1.password } };
```

Comment:
Shouldn't this be `$ne: null` based on the test title?

Reply:
yeah, but doc1.password is null actually.
Let me just explicitly write null here!